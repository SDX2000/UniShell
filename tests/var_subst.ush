#!/usr/bin/env unishell
echo "================"
echo $SCRIPT_NAME
echo "================"

set msg "Hi there"

echo $msg
echo ${msg}
echo "${msg}."

set hello hello
set world world

echo $hello $world
echo ${hello} ${world}
echo "$hello $world"
echo "${hello}X${world}."