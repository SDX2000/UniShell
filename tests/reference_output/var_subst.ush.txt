================
var_subst.ush
================
Hi there
Hi there
Hi there.
hello world
hello world
hello world
helloXworld.
